scripts/tools.sh

```diff
@@ -61,7 +61,7 @@ fi
 # air - code changes watcher
 if ! command -v air &> /dev/null
 then
-  go install github.com/air-verse/air@v1.52.3
+  go install github.com/air-verse/air@latest
```

Comment:
Please stabilize it to particular version, so we do not encounter any issues when not updating frequently